Prac05/state_names.py

```diff
@@ -0,0 +1,44 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+
+
+def main():
+    """This function is to execute the matching."""
+    print(CODE_TO_NAME)
+    for code, name in CODE_TO_NAME.items():
+        print(f"{code: <4} is {name}")
+
+    state_abbreviation = get_valid_input()
+    while state_abbreviation != "":
+        print(f"{state_abbreviation} is {CODE_TO_NAME[state_abbreviation]}")
+        state_abbreviation = get_valid_input()
+    print("This is the end of the comparison.")
+
+
+def get_valid_input():
+    """Deliver the error checking for input."""
+    state_abbreviation = ''
```

Comment:
unnecessary, you can create this var on line 31

Reply:
Interesting... my latest version python compiler suggest not to delete this.